include/scenefx/types/wlr_scene.h

```diff
@@ -140,6 +140,15 @@ struct wlr_scene_rect {
 	float color[4];
 };
 
+/** A scene-graph node displaying a shadow */
+struct wlr_scene_shadow {
+	struct wlr_scene_node node;
+	int width, height;
+	int corner_radius;
+	float color[4];
```

Comment:
Might make our lives a little easier if we use the wlr_color struct

Reply:
Similar to the rect struct we use it under the scenes

Reply:
Pun intended